util/dataset.go

```diff
@@ -32,13 +32,14 @@ var validTypes = map[string]string{
 }
 
 type DataSet struct {
-	ID             string   `config:"id" json:"id,omitempty" yaml:"id,omitempty"`
-	Title          string   `config:"title" json:"title" validate:"required"`
-	Release        string   `config:"release" json:"release"`
-	Type           string   `config:"type" json:"type" validate:"required"`
-	IngestPipeline string   `config:"ingest_pipeline,omitempty" config:"ingest_pipeline" json:"ingest_pipeline,omitempty" yaml:"ingest_pipeline,omitempty"`
-	Streams        []Stream `config:"streams" json:"streams,omitempty" yaml:"streams,omitempty" `
-	Package        string   `json:"package,omitempty" yaml:"package,omitempty"`
+	ID             string         `config:"id" json:"id,omitempty" yaml:"id,omitempty"`
+	Title          string         `config:"title" json:"title" validate:"required"`
+	Release        string         `config:"release" json:"release"`
+	Type           string         `config:"type" json:"type" validate:"required"`
+	IngestPipeline string         `config:"ingest_pipeline,omitempty" config:"ingest_pipeline" json:"ingest_pipeline,omitempty" yaml:"ingest_pipeline,omitempty"`
```

Comment:
Consider adding a godoc comment above this that starts with `// Deprecated: `.

Reply:
done, good idea.